api/deployment.go

```diff
@@ -46,7 +46,7 @@ import (
 //     description: Successfully created the deployment
 //     type: json
 //     schema:
-//       "$ref": "#/definitions/Build"
+//       "$ref": "#/definitions/Deployment"
```

Comment:
@meinenec I corrected this at the same time, I assume this was a copy-pasta error originally.